server/lib/generate-alerts/generate-alerts-script.js

```diff
@@ -310,6 +312,51 @@ function generateAlert(params) {
     alert.GeoLocation = randomArrayItem(GeoLocation);
   }
 
+  if (params.office) {
+    const beforeDate = new Date(new Date(alert.timestamp) - 3 * 24 * 60 * 60 * 1000);
+    const IntraID = randomArrayItem(Office.arrayUuidOffice);
+    const InterID = randomArrayItem(Office.arrayUuidOffice);
+    const OrgID = randomArrayItem(Office.arrayUuidOffice);
+    const objID = randomArrayItem(Office.arrayUuidOffice);
+    const userID = randomArrayItem(Office.arrayUuidOffice);
+    const appID = randomArrayItem(Office.arrayUuidOffice);
+
+    alert.rule = randomArrayItem(Office.arrayRulesOffice);
+    alert.decoder = randomArrayItem(Office.arrayDecoderOffice);
+    alert.GeoLocation = randomArrayItem(GeoLocation);
+    alert.data.integration = "Office365";
+    alert.location = Office.arrayLocationOffice;
+    alert.data.office365 = {
+      CreationTime: formatDate(beforeDate, 'Y-M-DTh:m:s.lZ'),
+      Id: IntraID,
+      Operation: "UserLoggedIn",
+      OrganizationId: OrgID,
+      RecordType: "15",
+      ResultStatus: "Success",
+      UserKey: userID,
+      UserType: "0",
+      Version: "1",
+      Workload: "AzureActiveDirectory",
+      ClientIP: "77.231.182.17",
+      ObjectId: objID,
+      UserId: "testing.account@wazuh.com",
+      AzureActiveDirectoryEventType: "1",
+      ExtendedProperties: Office.arrayExtendedPropertiesOffice,
+      ModifiedProperties: [],
+      Actor: Office.arrayActorOffice,
+      ActorContextId: OrgID,
+      ActorIpAddress: "77.231.182.17",
+      InterSystemsId: InterID,
+      IntraSystemId: IntraID,
+      Target: Office.arrayTargetOffice,
+      TargetContextId: OrgID,
+      ApplicationId: appID,
+      DeviceProperties: Office.arrayDevicePropertiesOffice,
```

Comment:
**issue**: There are multiple values that are arrays. Maybe you want to select one of them instead of all the array?

**suggestion**: Select one value. You could use `randomArrayItem` function to do this.

Reply:
In the example JSON file, there are some array values with some objects for each array.
This should be an objects array by default like:

`"DeviceProperties": [{
				"Name": "BrowserType",
				"Value": "Chrome"
			}, {
				"Name": "IsCompliantAndManaged",
				"Value": "False"
			}, {
				"Name": "SessionId",
				"Value": "2a1fb8c4-ceb6-4fa0-826c-3d43f87de897"
			}],`